src/util/triplea/mapXmlCreator/MapXmlCreator.java

```diff
@@ -303,7 +303,6 @@ public MapXmlCreator(final boolean testMode) {
       try {
         loadXmlFromFilePath(myFile.getAbsolutePath());
       } catch (SAXException | IOException | ParserConfigurationException e) {
-        // TODO Auto-generated catch block
         log(Level.SEVERE, "Default Map XML File could not be loaded from '" + myFile.getAbsolutePath() + "'.");
```

Comment:
Note, the level of this too high, should be an error, not severe. Severe is more like out of memory, or something that will kill the entire JVM. 

Also, the log( .. ) command takes an exception as a last argument, so then you do not need the e.printStackTrace() on the next line. You will also that way get the exception message printed out as well.